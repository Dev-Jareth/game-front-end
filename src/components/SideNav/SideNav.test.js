import React from 'react';
import Renderer from 'react-test-renderer';
import Enzyme, {shallow, mount, render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
Enzyme.configure({adapter: new Adapter()})
import {SideNav} from './';

describe('SideNav component', () => {
    it('should render', () => expect(Renderer.create(<SideNav/>)).toMatchSnapshot());
    it('should render links', () => {
        let links = [
            {
                to: '/',
                icon: 'house',
                exact: true,
                content: 'home'
            }, {
                to: '/fakePath',
                exact: false,
                content: 'Not real'
            }
        ]
        expect(shallow(<SideNav links={links}/>).find('SideNavLink').length).toBe(2);
    })
})