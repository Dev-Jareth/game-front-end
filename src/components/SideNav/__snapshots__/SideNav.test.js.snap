// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideNav component should render 1`] = `
<nav
  className="side-nav"
>
  <ul />
</nav>
`;
